applications/StructuralMechanicsApplication/custom_python/add_custom_response_functions_to_python.cpp

```diff
@@ -0,0 +1,63 @@
+// KRATOS  ___|  |                   |                   |
+//       \___ \  __|  __| |   |  __| __| |   |  __| _` | |
+//             | |   |    |   | (    |   |   | |   (   | |
+//       _____/ \__|_|   \__,_|\___|\__|\__,_|_|  \__,_|_| MECHANICS
+//
+//  License:        BSD License
+//	                license: structural_mechanics_application/license.txt
+//
+//  Main authors:    Armin Geiser
+//
+
+// System includes
+
+// External includes
+#include <boost/python.hpp>
+
+// Project includes
+#include "includes/define.h"
+#include "custom_python/add_custom_response_functions_to_python.h"
+
+//Utilities
+
+//Response Functions
+#include "custom_response_functions/response_utilities/response_function.h"
+#include "custom_response_functions/response_utilities/strain_energy_response_function.h"
+#include "custom_response_functions/response_utilities/mass_response_function.h"
+#include "custom_response_functions/response_utilities/eigenfrequency_response_function.h"
+#include "custom_response_functions/response_utilities/eigenfrequency_response_function_lin_scal.h"
+
+
+namespace Kratos
+{
+namespace Python
+{
+
+void  AddCustomResponseFunctionsToPython()
+{
+    using namespace boost::python;
```

Comment:
I guess the boost::python things here are now outdated and should be adjusted to the new pybind syntax.